sources/1-drawing/Bungee-Regular.ufo/lib.plist

```diff
@@ -4158,6 +4158,8 @@
     <integer>1</integer>
     <key>com.typemytype.robofont.compileSettings.checkOutlines</key>
     <integer>1</integer>
+    <key>com.typemytype.robofont.compileSettings.createDummyDSIG</key>
```

Comment:
Was there a reason for this change? I don't think it influences the build.

Reply:
This changed automatically when setting the new metrics in Robofont. It doesn't seem to influence the build so we can omit this commit.